File: src/Lession5/Ex4.java
package Lession5;

public class Ex4 {
    static void main() {
        int[] array_a = new int[] {3, 5, 7, 2, 8, 4, 6, 9, 1, 10};
        int[] array_b = new int[] {3, 5, 7, 2, -8, 4, 6, 9, 1, 10};

        System.out.println("Array A contains only positive numbers: " + checkForNegative(array_a));
        System.out.println("Array B contains only positive numbers: " + checkForNegative(array_b));
    }

    static boolean checkForNegative (int[] arr) {
        for (int k = 0; k < arr.length; k++) {
            if (arr[k] < 0) return false;
        }
        return true;
    }

}

File: src/Lession5/Ex4.kt
package Lession5

fun main() {
    val array_a = intArrayOf(7, 14, 3, 19, 8, 12, 5, 17, 2, 11, 9, 16, 4, 20, 6, 13, 1, 18, 10, 15)
    val array_b = intArrayOf(-7, 14, 3, 19, 8, 12, 5, 17, 2, 11, 9, 16, 4, 20, 6, 13, 1, 18, 10, 15)

    println("Array A contains only positive numbers: ${checkForPositive(array_a)}")
    println("Array B contains only positive numbers: ${checkForPositive(array_b)}")
}

fun checkForPositive(arr: IntArray): Boolean {
    for (a in arr) {
        if (a < 0) {
            return false
        }
    }
    return true
}
